### DISCUSSION TITLE
Flaky connection with error "dnsforward: invalid msg type HTTPS for custom IP blocking mode"

### DISCUSSION THREAD

**Discussion by jayjoshi64** [AdguardTeam/AdGuardHome#7104] [General]
I am getting the following error in the docker logs 
```
[error] dnsforward: invalid msg type HTTPS for custom IP blocking mode
```

when that happens, internet stops working for all devices.  Any idea what might be causing it?  
btw, I added an ip address in blocking list since I wanted to bypass dns lookup in that device, is that causing this issue?

**Comment by evilalmus** [comment]
I'm having this same issue except I didn't change anything.

**Comment by jayjoshi64** [comment]
By any chance you have configured custom ip address for blocking domains?   I wanted to show a custom page when a request is blocked, so I configured it. That seems to have caused this. Disabling that helped.

**Reply by scyto** [reply]
> By any chance you have configured custom ip address for blocking domains? I wanted to show a custom page when a request is blocked, so I configured it. That seems to have caused this. Disabling that helped.

Why yes, i had (as part of troubleshooting another issue flooding the log), setting back to default did indeed seem to fix this, thanks!
